### ISSUE TITLE
Request: show the embedded player while in "reading" mode

### ISSUE THREAD

**Issue by nandocova** [jemstelos/obsidian-media-notes#40]
The plugin works just fine so far, but if I switch obsidian to the "reading" view, the embedded player closes. It would be really useful to keep it open.

**Comment by 22sh22** [comment]
I agree, and I think it would be much more better if it supports **"tab or window view".**
the editor automatically moves to right or upward whenever I type something in media link.
Setting that makes not to open video automatically, and to provide manual button to open the viewer window.
that's the reason why I didn't picked this plugin up, but other plugins like media extended is quite complicated and weird because of its UI.
 This plugin would become the best when these problems get solved! I'll wait for it until my Second brain get finished!

p.s ( i hope it supports the templater, too (even though I found an unofficial way)
